db/schema/05_menu_item_categories.sql.sql

```diff
@@ -0,0 +1,8 @@
+DROP TABLE IF EXISTS widgets CASCADE;
```

Comment:
Should it be `DROP TABLE IF EXIST menu_item_categories CASCADE` ?

Reply:
good catch Sam! I will adjust it